client/files.go

```diff
@@ -575,28 +575,30 @@ func (client *Client) Pull(opts *PullOptions) error {
 	if mediaType != "multipart/form-data" {
 		// If it's not multipart, it's likely an API error, so try parsing it as such.
 		var serverResp response
-		err := decodeInto(resp.Body, &serverResp)
-		if err != nil {
+		if err := decodeInto(resp.Body, &serverResp); err != nil {
```

Comment:
Let's please avoid making stylistic changes like this in this PR (here and elsewhere). It makes the actual changes harder to review.